1227-customer_registry/574-application_administrator_opens_a_customer_institution.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Application Administrator opens a Customer Institution
+
+    @574
+    Scenario Outline: Application Administrator opens a Customer Institution
+        Given that the user is logged in as Application Administrator
+        When they open a Customer Institution
+        Then they see fields:
+            | Name in organization registry |
+            | Display name                  |
+            | Short display name            |
+            | Feide Organization ID         |
+        And they can click Save to persist values in fields
+        And they can add and remove Institution Administrator(s)
```

Comment:
```suggestion
        And they see buttons Add and Remove
```

Reply:
Evt skrive `And they see Buttons Save, Add and Remove` ? eller `And they see <Button>` + eksempelliste?

Reply:
burde egentlig ha med opplisting av admins også... feks
```
(...)
And they see the Save button
And they see list of (Institution) Administrators for the Customer Institution
And every (Institution) Administrator has a Remove button
And they see button to add a new (Institution) Administrator
```